src/pages/tokens/[id].vue

```diff
@@ -48,8 +48,20 @@ const { fetchTokenDetails } = tokenDetailsStore
 
 const { isLoading } = useLoading()
 
-await fetchTokenDetails(route.params.id)
-
+try {
+  await fetchTokenDetails(route.params.id)
+} catch (error) {
+  if ([400, 404].includes(error.response?.status)) {
```

Comment:
What happens if the status is 500? It also applies to the other details screens.

Reply:
Good catch, `throw error` is missing to show UnexpectedError page